internal/apps/apps.go

```diff
@@ -7,6 +7,11 @@ import (
 	"strings"
 )
 
+var (
+	reFileID = regexp.MustCompile("(storage:(//)?)?(?P<fileID>[a-f0-9]{8}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{12})$")
+	reFilePattern = regexp.MustCompile("^(storage:filename=)(?P<filename>[\\S][\\S ]+(\\.ipa|\\.apk))$")
```

Comment:
What of the `.aab` files?

Reply:
Those regex comes from RDC, `.aab` files will be rejected from the `storage:filename=` synthax when coming to RDC.
I made it work similar way until this is fixed on their side (and then we will be able to fix it here).